### PR TITLE
fixes #2838 - taxonomies controller - fix in params and finder

### PR DESCRIPTION
**Pull request by tstrachota**
- added mising apipie params for search & pagination
- simplified resource finder
- now finds also by name
- show response has timestamps
- fix resource names in apipie docs

### CODE REVIEW

**Comment by domcleal** [comment]
Could you review please @isratrade?

File: app/controllers/concerns/api/v2/taxonomies_controller.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ module Api::V2::TaxonomiesController
   end
 
   api :GET, '/:resource_id', 'List all :resource_id'
+  param :search, String, :desc => "filter results"
+  param :order, String, :desc => "sort results"
+  param :page, String, :desc => "paginate results"
+  param :per_page, String, :desc => "number of entries per request"
   def index
     if @nested_obj
       #@taxonomies = @domain.locations.paginate(paginate_options)
```
Comment (isratrade) [inline app/controllers/concerns/api/v2/taxonomies_controller.rb]: All looks good.  Just one nitpick.  Add .search_for to the comment to match as an example of what is generated.

**Comment by mbacovsky** [comment]
I fixed the comment, is it okay to merge now?

**Comment by domcleal** [comment]
Nice improvements, thanks @tstrachota and @mbacovsky.  Merged as 0bed6e7.

**Pull request closed without merging**